.travis.yml

```diff
@@ -10,6 +10,10 @@ virtualenv:
   system_site_packages: true
 
 before_install:
+<<<<<<< HEAD
+  - pip install mpmath
```

Comment:
@lidavidm Yes, what should I do here? Should I remove the `HEAD` and `master` lines? Or what should I do? @lidavidm Could you help me on this?